hw/ip/otbn/dv/verilator/otbn_top_sim.core

```diff
@@ -0,0 +1,67 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:ip:otbn_top_sim:0.1"
+description: "Standalone OpenTitan Big Number Accelerator (OTBN) simulation"
+
+filesets:
+  files_otbn:
+    depend:
+      - lowrisc:ip:otbn:0.1
+  files_verilator:
+    depend:
+      - lowrisc:dv_verilator:memutil_verilator
+      - lowrisc:dv_verilator:simutil_verilator
+    files:
+      - otbn_top_sim.cc: { file_type: cppSource }
+      - otbn_top_sim.sv: { file_type: systemVerilogSource }
+  files_verilator_waiver:
+    files:
+      - otbn_top_sim_waivers.vlt
+    file_type: vlt
+
+targets:
+  default: &default_target
+    filesets:
+      - files_verilator_waiver
+      - files_otbn
+      - files_verilator
+    toplevel: otbn_top_sim
+
+  lint:
+    <<: *default_target
+    default_tool: verilator
+    tools:
+      verilator:
+        mode: lint-only
+        verilator_options:
+          - "-Wall"
+          # RAM primitives wider than 64bit (required for ECC) fail to build in
+          # Verilator without increasing the unroll count (see Verilator#1266)
+          - "--unroll-count 72"
+
+  sim:
+    <<: *default_target
+    default_tool: verilator
+    tools:
+      vcs:
+        vcs_options:
+          - '-xlrm uniq_prior_final'
+          - '-debug_access+r'
+      verilator:
+        mode: cc
+        verilator_options:
+          # Disabling tracing reduces compile times but doesn't have a
+          # huge influence on runtime performance.
+          - '--trace'
+          - '--trace-fst' # this requires -DVM_TRACE_FMT_FST in CFLAGS below!
+          - '--trace-structs'
+          - '--trace-params'
+          - '--trace-max-array 1024'
+          - '-CFLAGS "-std=c++11 -Wall -DVM_TRACE_FMT_FST -DTOPLEVEL_NAME=otbn_top_sim -g"'
```

Comment:
Do we actually need -g? If we're running GDB on the verilated binary, I think something has gone quite wrong.

Reply:
I don't think so, this is just blind copy paste from another .core so happy to remove it